test/proto/BUILD

```diff
@@ -11,6 +11,20 @@ envoy_package()
 
 exports_files(["bookstore.proto"])
 
+# The binary proto descriptors are generated and stored prior to runtime. The command to generate
+# proto descriptors from the command line can be found here: https://lyft.github.io/envoy/docs/conf
+# iguration/http_filters/grpc_json_transcoder_filter.html#config-http-filters-grpc-json-transcoder.
+# Alternatively, the proto descriptors can be generated with bazel rule envoy_proto_descriptor. The
```

Comment:
I see. I didn't generate the descriptors directly via CLI. Rather, I generated them indirectly via the bazel rule.